token_check.tf

```diff
@@ -0,0 +1,12 @@
+resource "null_resource" "token_check" {
+  triggers = {
+    keeper = base64sha256("${jsonencode(var.secure_strings)}-${var.configuration}")
+  }
+
+  lifecycle {
+    precondition {
+      condition     = length(keys(var.secure_strings)) > 0 || var.configuration != ""
+      error_message = "Either var.secure_strings or var.configuration must be set"
```

Comment:
Are you sure? What if they want to export other variables, or just do some other stuff during startup?

Reply:
This will allow them to use both at the same time. But at least one must be set.

I was also thinking, if we do release 3.0.0 we _could_ just make this a breaking change and make the secure_strings required and configuration is optional. Force folks to switch to this new, more secure, way.